package/libpiksi/libpiksi/src/networking.c

```diff
@@ -143,10 +135,7 @@ static s32 fill_network_state_struct(struct ifaddrs *ifa, void *userdata)
 void query_network_state(msg_network_state_resp_t *interfaces, u8 interfaces_n, u8 *returned_interfaces)
 {
   struct fill_network_state_s fill_data = {
-    .interfaces = interfaces,
-    .interfaces_size = interfaces_n,
-    .interfaces_count = returned_interfaces
-  };
+    .interfaces = interfaces, .interfaces_size = interfaces_n, .interfaces_count = returned_interfaces};
```

Comment:
:-O

Reply:
I found [this](https://reviews.llvm.org/D33029) and [this](https://reviews.llvm.org/D34238) which seem to indicate a successful workaround is to simply put a comma after all initializer arguments